Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/core/ConceptTypesService.java

```diff
@@ -0,0 +1,24 @@
+package edu.asu.conceptpower.servlet.core;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class ConceptTypesService {
+    
+    public enum ConceptTypes {
+        GENERIC_WORDNET_CONCEPT, SPECIFIC_WORDNET_CONCEPT, LOCAL_CONCEPT
+    }
+    
+    public ConceptTypes getConceptTypeByConceptId(String id) {
+        if (id == null) {
+            return null;
+        }
+        if (id.contains("??")) {
+            return ConceptTypes.GENERIC_WORDNET_CONCEPT;
+        } else if (id.startsWith("WID")) {
```

Comment:
you don't need the else here, as the code only continues if id does not contain '??' (otherwise it'll return)